upload/admin/controller/extension/module/smaily_for_opencart.php

```diff
@@ -456,10 +456,15 @@ public function index() {
             $data['rss_sort_order'] = $this->config->get('smaily_for_opencart_rss_sort_order');
         }
         // RSS product limit.
-        if (isset($this->request->post['smaily_for_opencart_rss_limit'])) {
+        if (! empty($this->request->post['smaily_for_opencart_rss_limit'])) {
             $data['rss_limit'] = $this->request->post['smaily_for_opencart_rss_limit'];
         } else {
-            $data['rss_limit'] = $this->config->get('smaily_for_opencart_rss_limit');
+            if(! empty($this->config->get('smaily_for_opencart_rss_limit'))) {
+                $data['rss_limit'] = $this->config->get('smaily_for_opencart_rss_limit');
+            } else {
+                $data['rss_limit'] = 50;
+            }
+
```

Comment:
You can write this using following pattern:

```php
if (...) {
  // ...
} else if (...) {
  // ...
} else {
  // ...
}
```

This way you will reduce unnecessary indentation.

Reply:
added 
1d67064